src/simulations/spatial_parameters.jl

```diff
@@ -280,39 +281,59 @@ function default_spatially_varying_soil_parameters(
 
     x = parent(K_sat)
     μ = mean(log10.(x[x .> 0]))
-    K_sat .= oceans_to_value.(K_sat, soil_params_mask, 10.0^μ)
+    K_sat .= masked_to_value.(K_sat, soil_params_mask, 10.0^μ)
 
-    ν .= oceans_to_value.(ν, soil_params_mask, 1)
+    ν .= masked_to_value.(ν, soil_params_mask, 1)
 
-    θ_r .= oceans_to_value.(θ_r, soil_params_mask, 0)
+    θ_r .= masked_to_value.(θ_r, soil_params_mask, 0)
 
 
     S_s =
-        oceans_to_value.(
+        masked_to_value.(
             ClimaCore.Fields.zeros(subsurface_space) .+ FT(1e-3),
             soil_params_mask,
             1,
         )
-    ν_ss_om =
-        oceans_to_value.(
-            ClimaCore.Fields.zeros(subsurface_space) .+ FT(0.1),
-            soil_params_mask,
-            0,
-        )
-    ν_ss_quartz =
-        oceans_to_value.(
-            ClimaCore.Fields.zeros(subsurface_space) .+ FT(0.1),
-            soil_params_mask,
-            0,
-        )
-    ν_ss_gravel =
-        oceans_to_value.(
-            ClimaCore.Fields.zeros(subsurface_space) .+ FT(0.1),
-            soil_params_mask,
-            0,
+
+    soilgrids_artifact_dir =
+        ClimaLand.Artifacts.soil_grids_params_artifact_folder_path(;
+            lowres = true,
+            context,
         )
-    PAR_albedo = ClimaCore.Fields.zeros(surface_space) .+ FT(0.2)
-    NIR_albedo = ClimaCore.Fields.zeros(surface_space) .+ FT(0.2)
+    soilgrids_artifact_path = joinpath(soilgrids_artifact_dir, "soil_solid_vol_fractions_soilgrids_lowres.nc")
+    ν_ss_om = SpaceVaryingInput(
+        soilgrids_artifact_path,
+        "nu_ss_om",
+        subsurface_space;
+        regridder_type,
+        regridder_kwargs = (; extrapolation_bc,),
+    )
+
+    ν_ss_quartz = SpaceVaryingInput(
+        soilgrids_artifact_path,
+        "nu_ss_sand",
+        subsurface_space;
+        regridder_type,
+        regridder_kwargs = (; extrapolation_bc,),
+    )
+
+    ν_ss_gravel = SpaceVaryingInput(
+        soilgrids_artifact_path,
+        "nu_ss_cf",
+        subsurface_space;
+        regridder_type,
+        regridder_kwargs = (; extrapolation_bc,),
+    )
+    # we require that the sum of these be less than 1 and equal to or bigger than zero.
+    # The input should satisfy this almost exactly, but the regridded values may not.
+    function texture_sum_norm(ν_ss_gravel, ν_ss_quartz, ν_ss_om)
+        texture_sum = ν_ss_gravel + ν_ss_quartz + ν_ss_om 
+        texture_norm = texture_sum <= 1 ? 1 : texture_sum
+    end
+    @. ν_ss_gravel = ν_ss_gravel / texture_sum_norm(ν_ss_gravel, ν_ss_quartz, ν_ss_om)
+    @. ν_ss_om = ν_ss_om / texture_sum_norm(ν_ss_gravel, ν_ss_quartz, ν_ss_om)
+    @. ν_ss_quartz = ν_ss_quartz / texture_sum_norm(ν_ss_gravel, ν_ss_quartz, ν_ss_om)
```

Comment:
If the sum of the three relative volumes is greater than 1, then value returned by `texture_sum_norm(ν_ss_gravel, ν_ss_quartz, ν_ss_om)` will be different for each of the 3 calls, and the rescaling will be less for `ν_ss_quartz` than for `ν_ss_om`, which will be less than `ν_ss_gravel`. The final result will then still sum to greater than 1. Is this intentional?

Reply:
good catch!! I need to compute the sum and store it, and then use it for all three computations.